src/Query/User/PackageQuery/DbalPackageQuery.php

```diff
@@ -29,10 +30,42 @@ public function findAll(string $organizationId, int $limit = 20, int $offset = 0
         return array_map(function (array $data): Package {
             return $this->hydratePackage($data);
         }, $this->connection->fetchAll(
-            'SELECT id, organization_id, type, repository_url, name, latest_released_version, latest_release_date, description, last_sync_at, last_sync_error, webhook_created_at
-            FROM "organization_package"
-            WHERE organization_id = :organization_id
-            ORDER BY name ASC
+            'SELECT
+                p.id,
+                p.organization_id,
+                p.type,
+                p.repository_url,
+                p.name,
+                p.latest_released_version,
+                p.latest_release_date,
+                p.description,
+                p.last_sync_at,
+                p.last_sync_error,
+                p.webhook_created_at,
+                (
+                    SELECT s.status
+                    FROM organization_package_scan_result s
+                    WHERE s.package_id = p.id
+                    ORDER BY date DESC
+                    LIMIT 1
+                ) scan_result_status,
+                (
+                    SELECT s.date
+                    FROM organization_package_scan_result s
+                    WHERE s.package_id = p.id
+                    ORDER BY date DESC
+                    LIMIT 1
+                ) scan_result_date,
+                (
+                    SELECT s.content
+                    FROM organization_package_scan_result s
+                    WHERE s.package_id = p.id
+                    ORDER BY date DESC
+                    LIMIT 1
+                ) scan_result_content
```

Comment:
why not left join? then if these columns are not present then just nulls will be returned

Reply:
It bugs me also but If we join, grouping by those columns or agreggation is required and one of those columns is json which can't be done (not comparable)

Reply:
Well, probably an additional query will be better